src/dom-wrappers.ts

```diff
@@ -13,7 +13,7 @@ import { Registry, TramOneHTMLElement, TramOneSVGElement } from './types';
  * @return tagged template function that builds HTML components
  */
 export const registerHtml = (registry?: Registry) => {
-	return registerDom<TramOneHTMLElement>(null, registry);
+	return registerDom<TramOneHTMLElement>(null, { ...registry, '': fragment });
```

Comment:
we're including fragment as a default element in the registry, this means that (like React) it's included by default!